### PR TITLE
add docker example

### PR DESCRIPTION
**Pull request by tarekbadrshalaan**
I tried to create an example how to use docker with `confluent-kafka-go`
please feel free to ask for update.

### CODE REVIEW

**Comment by tarekbadrshalaan** [comment]
for some reason sign our **Contributor License Agreement** not working, i tried to submit mutable times.

File: examples/docker_example/go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/tarekbadrshalaan/GoKafka
```
Comment (edenhill) [inline examples/docker_example/go.mod]: This should be `github.com/confluentinc/confluent-kafka-go/examples/docker_example`

Comment (tarekbadrshalaan) [reply examples/docker_example/go.mod]: thanks for your comment @edenhill , the pull request updated now.

File: examples/docker_example/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+RUN go mod download
+
+COPY . .
+
+RUN go build -o main
+
+# final stage
+FROM ubuntu
+
+COPY --from=builder /usr/lib/pkgconfig /usr/lib/pkgconfig
```
Comment (edenhill) [inline examples/docker_example/Dockerfile]: I dont think the pkgconfig files are needed after the build is complete.

File: examples/docker_example/go.mod [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module github.com/confluentinc/confluent-kafka-go/examples/docker_example
+
+require github.com/confluentinc/confluent-kafka-go v0.11.6
```
Comment (edenhill) [inline examples/docker_example/go.mod]: bump to v1.0.0

File: examples/docker_example/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# build stage
+FROM golang as builder 
+
+# librdkafka Build from source
+
+RUN git clone https://github.com/edenhill/librdkafka.git
+
+WORKDIR librdkafka
+
+RUN ./configure --prefix /usr
```
Comment (edenhill) [inline examples/docker_example/Dockerfile]: add --install-deps

File: examples/docker_example/go.sum [hunk 5]
```diff
@@ -0,0 +1,2 @@
+github.com/confluentinc/confluent-kafka-go v0.11.6 h1:rEblubnNXCjRThwAGnFSzLKYIRAoXLDC3A9r4ciziHU=
```
Comment (edenhill) [inline examples/docker_example/go.sum]: bump to v1.0.0

File: examples/docker_example/go.sum [hunk 6]
```diff
@@ -0,0 +1,2 @@
...
+github.com/confluentinc/confluent-kafka-go v0.11.6/go.mod h1:u2zNLny2xq+5rWeTQjFHbDzzNuba4P1vo31r9r4uAdg=
```
Comment (edenhill) [inline examples/docker_example/go.sum]: dito

File: examples/docker_example/main.go [hunk 7]
```diff
@@ -0,0 +1,34 @@
+package main
```
Comment (edenhill) [inline examples/docker_example/main.go]: Add license header, copy it from one of the other examples and change the year to 2019

**Comment by edenhill** [comment]
Review comments not addressed, closing for now.

Thanks for your contribution!

**Pull request closed without merging**